AdminWebsite/AdminWebsite/ClientApp/src/app/bookings-list/booking-details/booking-details.component.ts

```diff
@@ -221,6 +231,7 @@ export class BookingDetailsComponent implements OnInit, OnDestroy {
         this.$subscriptions.push(
             this.videoHearingService.getHearingById(this.hearingId).subscribe(
                 newData => {
+                    newData.status = BookingStatus.Created;
```

Comment:
What is this doing?

Reply:
ah, hangover from manual testing. will remove that